content/2017.06.kinto-elasticsearch.rst

```diff
@@ -0,0 +1,249 @@
+Kinto and ElasticSearch
+#######################
+
+:lang: en
+:date: 2017-06-21
+:url: kinto-elasticsearch
+:summary: Introducing our ElasticSearch plugin for Kinto
+
+
+ElasticSearch integration
+=========================
+
+Kinto is a generic API on top of a storage backend.
+ElasticSearch is a generic API on top of an indexing backend. When combining both,
+your data is stored in a persistent database, and you can search, aggregate and
+filter it efficiently.
+
+This integration has been sitting around for a long time, since it was the matter of
+our `Kinto plugin tutorial <http://kinto.readthedocs.io/en/7.0.1/tutorials/write-plugin.html>`_! But recently, we had a concrete use-case at Mozilla
+so we published it as `an official plugin <https://github.com/Kinto/kinto-elasticsearch/>`_ :)
+
+
+Use case
+========
+
+Mozilla publishes its products for several platforms, in a lot of different locales (*70-100 regions/languages*),
+and at different stages of maturity (*nightly, beta, release, ESR*). And currently there
+is no comprehensive or official database for every released version and its associated
+metadata (revision, build date, ...).
+
+Our idea was to build an API to allow third-party scripts and tools to publish or query
+metadata about released products. As well as a nice UI to explore and browse the catalog.
+
+Kinto gives us a ready-to-use JSON storage API, whose accessibility via ``curl`` is rather intuitive,
+and for which we can manage authentication and permissions using simple tokens or LDAP.
+ElasticSearch gives us a rocket fast index that powers the facetted search of our catalog.
```

Comment:
I would write `faceted`, but I just looked it up and apparently `facetted` is acceptable and British.